src/Socket.h

```diff
@@ -263,7 +263,7 @@ class Socket
      * \param fromlen    Optional, only required if 'from' is given: length of from struct
      * \return    Number of bytes received or SOCKET_ERROR
      */
-    int recvfrom ( char* data, const int buffersize, struct sockaddr* from = NULL, socklen_t* fromlen = NULL) const;
+    int recvfrom ( char* data, const int buffersize, struct sockaddr* from = nullptr, socklen_t* fromlen = nullptr) const;
```

Comment:
Spacing around function and arguments is different. Oh, it's just this file. Maybe a future tidy up.

Reply:
I didn't clang-format  that one, Socket.cpp is mainly used by v4 timeshift code (which needs tidying too, and I don't use it or support it so I won't be able to test or support any changes.

Reply:
I’ve never had clang format introduce a bug on me so should be fine to do so (if you wish). It’s kind of the point of the tool. It makes no functional changes to your code.

Reply:
I don't disagree but if for anything to do with v4 timeshift I leave it alone so I can tell users the code hasn't change when it inevitably fails.